assets/base/i18n/en.json

```diff
@@ -125,6 +125,7 @@
   "mobile.account_notifications.threads_start": "Threads that I start",
   "mobile.account_notifications.threads_start_participate": "Threads that I start or participate in",
   "mobile.account.settings.save": "Save",
+  "mobile.account.settings.cancel": "Cancel",
```

Comment:
Non blocking: We can create a generic string for `cancel` link something like `mobile.general.cancel` so we can use it across app for different uses cases. We have been creating new translations for generic strings which means making the translators adding them again for all translations.

@enahum @hmhealey ☝️

Reply:
0/5

Reply:
I am good with the change here. 👍 

@enahum I wasn't talking about the change for this PR but more of a general approach for translations. I don't know estimate of the work involved in making a new translation but if it can be avoided then i think it is better as it even keeps the files cleaner.

Reply:
I would agree that text snippets that are repeated in several places and aren't likely to diverge from each other should be considered for a global, single translation. I also agree that it isn't part of this PR. :)